sequencing-server/src/app.ts

```diff
@@ -156,64 +156,61 @@ app.get('/health', (_: Request, res: Response) => {
 
 app.post('/put-dictionary', async (req, res, next) => {
   const dictionary = req.body.input.dictionary as string;
-  const type = req.body.input.type as string;
   logger.info(`Dictionary received`);
 
-  let parsedDictionary: CommandDictionary | ChannelDictionary | ParameterDictionary;
-  let dictionaryPath: string = '';
-  switch (type) {
-    case DictionaryType.COMMAND: {
-      parsedDictionary = ampcs.parse(dictionary, null, { ignoreComment: true });
-      break;
+  let parsedDictionaries : {
+    commandDictionary?: CommandDictionary,
+    channelDictionary?: ChannelDictionary,
+    parameterDictionary?: ParameterDictionary
+  };
+  if (pluginManager.hasPlugin(getEnv().DICTIONARY_PARSER_PLUGIN) && pluginManager.getPlugin(getEnv().DICTIONARY_PARSER_PLUGIN).parseDictionary){
```

Comment:
Could we add a small route that returns which parser plugin is currently loaded? This could be surfaced via the UI, so users get some insight into which dictionaries are able to be uploaded for a given deployment.